### ISSUE TITLE
Request: setting to change the green top bar color

### ISSUE THREAD

**Issue by hynow** [mhoeher/opentodolist#69]
The green is a little intense!  I'm glad it's not blue, but wish it could be set like the backgrounds.  I like the lilac color option.

**Comment by mhoeher** [comment]
Yeah... the green tone is basically on of the standard tones from the Google Material Theme (which the app implements across all platforms) and yes, it is pretty strong.

I could imagine two possible ways out of this:

- One would be - as you suggested - to introduce a setting that lets the user select between different color themes within the app.
- The other option would be to not use one of the standard colours from the Material palette but instead use "OpenTodoList green" (this is the one used in the app icon).

I personally have a slight bias towards the second option - simply because it somehow keeps a kind of consistent and uniform style to the app. Just for comparison, this is the current titlebar color used:

<img width="489" alt="Screenshot 2022-05-22 at 14 32 37" src="https://user-images.githubusercontent.com/1911135/169695312-bcf8b7c7-b034-4ee8-908c-7d020bbeec5c.png">

While this would be the "OpenTodoList green":

<img width="445" alt="Screenshot 2022-05-22 at 14 30 47" src="https://user-images.githubusercontent.com/1911135/169695227-f3af28e7-6e64-494e-91d1-22b5a2c1bacd.png">

As you can see, it is a little more muted and less bright - and hence, it probably would look less intense when used across the entire titlebar.

What do you think?
